### PR TITLE
docs(README): Clarify installation of dependencies

### PR DESCRIPTION
**Pull request by tvthatsme**
**What**:
Correct the installation instructions

**Why**:
The instructions previously stated that `react-ideal-image` should be installed as a devDependency.

**How**:
Changed dependency and added a note for the peerDependencies that are required.

**Checklist**:

* [x] Documentation
* [ ] Tests
* [x] Ready to be merged 
* [ ] Added myself to contributors table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,12 +44,15 @@ Read the [introduction](introduction.md).
...
 This module is distributed via [npm][npm] which is bundled with [node][node] and
-should be installed as one of your project's `devDependencies`:
+should be installed as one of your project's `dependencies`:
 
 ```
-npm install react-ideal-image react-waypoint --save
+npm install react-ideal-image --save
 ```
 
+> This package also depends on `react`, `prop-types`, and `react-waypoint`.
```
Comment (stereobooster) [inline README.md]: Why not simply `npm install react-ideal-image react-waypoint prop-types --save`?

Comment (tvthatsme) [reply README.md]: Either works I guess 😄 I am seeing that you draw a lot of inspiration from Kent C. Dodds and his Downshift project [does it that way](https://github.com/paypal/downshift/blob/master/README.md#installation).

Happy to change it to what you suggest though.

Comment (stereobooster) [reply README.md]: I guess both ways ok. I just thought it is easier to copy/paste `npm install ...` to get started

**Comment by stereobooster** [comment]
Thank you

**Pull request merged**